shared/chat/conversation/input-area/channel-mention-hud/mention-hud-container.js

```diff
@@ -0,0 +1,28 @@
+// @flow
+import {MentionHud} from '.'
+import {connect, type TypedState} from '../../../../util/container'
+import * as Constants from '../../../../constants/chat2'
+
+const mapStateToProps = (state: TypedState, {filter, conversationIDKey}) => {
```

Comment:
🍳 This is getting triggered on ad-hoc convos with a long list of just `#`:

<img width="700" alt="image" src="https://user-images.githubusercontent.com/11968340/36922083-bc1f14ce-1e34-11e8-90f5-ce77e01711ed.png">

Reply:
fixed in 10: https://github.com/keybase/client/pull/10755/files#diff-70f2ef2567d7de29de82025ea06ec482R17